Transpile bug

error TS6059: File '/home/mikhail/projects/scout-backend/jest.config.ts' is not under 'rootDir' '/home/mikhail/projects/scout-backend/src'. 'rootDir' is expected to contain all source files.
<!-- Edit the body of your new issue then click the ✓ "Create Issue" button in the top right of the editor. The first line will be the issue title. Assignees and Labels follow after a blank line. Leave an empty line before beginning the body of the issue. -->

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -98,5 +98,5 @@
     // "skipDefaultLibCheck": true,                      /* Skip type checking .d.ts files that are included with TypeScript. */
     "skipLibCheck": true                                 /* Skip type checking all .d.ts files. */
   },
-  "exclude": ["src/**/*.test.ts"]
+  "exclude": ["src/**/*.test.ts","jest.config.ts"]
 }

